app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
```

Comment:
- [ ] Your routes are trying to authenticate with Devise for each API endpoint which redirects to the sign in page. I would recommend you skip the `authenticate_user!` action to allow requests to the API controllers.

  ```
  class Api::V1::CommentsController < ApplicationController
    skip_before_action :authenticate_user!
    ...
  ```